vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx

```diff
@@ -45,108 +64,146 @@ module IntegersAndNumbers =
     /// This is a list of all tuples containing all the numbers from 0 to 99 and their squares.
     let sampleTableOfSquares = [ for i in 0 .. 99 -> (i, i*i) ]
 
-    // The next line prints a list that includes tuples, using %A for generic printing
+    // The next line prints a list that includes tuples, using %A for generic printing.
     printfn "The table of squares from 0 to 99 is:\n%A" sampleTableOfSquares
 
 
 /// Much of F# programming consists of defining functions that transform input data to produce
-/// useful resutls.
+/// useful results.
+///
+/// To learn more, see: https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/functions/
 module BasicFunctions = 
 
-    /// You use 'let' to define a function that accepts an integer argument and returns an integer. 
-    /// Parentheses are optional for function arguments,
+    /// You use 'let' to define a function. This one accepts an integer argument and returns an integer. 
+    /// Parentheses are optional for function arguments, except for when you use an explicit type annotation.
     let sampleFunction1 x = x*x + 3
 
     /// Apply the function, naming the function return result using 'let'. 
     /// The variable type is inferred from the function return type.
     let result1 = sampleFunction1 4573
+
+    // This line uses '%d' to print the result as an integer. This is type-safe.
+    // If 'result1' were not of type 'int', then the line would fail to compile.
     printfn "The result of squaring the integer 4573 and adding 3 is %d" result1
 
-    /// When needed, annotate the type of a parameter name using '(argument:type)'
+    /// When needed, annotate the type of a parameter name using '(argument:type)'.  Parentheses are required.
     let sampleFunction2 (x:int) = 2*x*x - x/5 + 3
 
     let result2 = sampleFunction2 (7 + 4)
     printfn "The result of applying the 1st sample function to (7 + 4) is %d" result2
 
     /// Conditionals use if/then/elid/elif/else.
     ///
-    /// Note that F# uses whitespace indentation-aware syntax like Python.
+    /// Note that F# uses whitespace indentation-aware syntax, similar to languages like Python or Ruby.
```

Comment:
IIRC Ruby is not whitespace sensitive - then have things like end keywords. http://stackoverflow.com/a/17052327/145701